examples/ina219_simpletest.py

```diff
@@ -7,7 +7,7 @@
 
 i2c_bus = board.I2C()
 
-ina219 = INA219(i2c_bus)
+ina219 = INA219(i2c_bus=i2c_bus, addr=0x40)
```

Comment:
Why add `i2c_bus=i2c_bus`? It is a positional argument and shouldn't need the `i2c_bus=` part.

Reply:
I'm used to using named parameters instead of positional ones.